x-pack/docs/en/ilm/pause-resume-ilm.asciidoc

```diff
@@ -1,4 +1,169 @@
 [[pause-resume-ilm]]
 == Pause and resume {ilm}
-ILM can be skipped on a per-index basis
-ILM as a whole can be placed in maintenance mode
+
+All indices that are managed by ILM will continue to run through
+their policies. There may be times when this is not desired on certain
+indices, or maybe even all the indices in a cluster. For example,
+maybe there are scheduled maintenance windows when cluster topology
+changes are desired that may impact running ILM actions. For this reason,
+ILM has two ways to disable operations.
+
+Normally, ILM will be running by default.
+To see the current operating status of ILM, use the <<ilm-get-status,Get Status API>>
+to see the current state of ILM.
+
+////
+[source,js]
+--------------------------------------------------
+PUT _ilm/my_policy
+{
+  "policy": {
+    "phases": {
+      "warm": {
+        "minimum_age": "10d",
+        "actions": {
+          "forcemerge": {
+            "max_num_segments": 1
+          }
+        }
+      },
+      "delete": {
+        "minimum_age": "30d",
+        "actions": {
+          "delete": {}
+        }
+      }
+    }
+  }
+}
+
+PUT my_index
+{
+  "settings": {
+    "index.lifecycle.name": "my_policy"
+  }
+}
+--------------------------------------------------
+// CONSOLE
+////
+
+[source,js]
+--------------------------------------------------
+GET _ilm/status
+--------------------------------------------------
+// CONSOLE
+
+If the request does not encounter errors, you receive the following result:
+
+[source,js]
+--------------------------------------------------
+{
+  "operation_mode": "RUNNING"
+}
+--------------------------------------------------
+// CONSOLE
+// TESTRESPONSE
+
+The operating modes of ILM:
+
+
+.ILM Operating Modes
+|===
+|Name |Description
+|RUNNING |Normal operation where all policies are executed as normal
+|STOPPING |This represents a state where only sensitive actions (like the Shrink Action) will be executed
```

Comment:
I think calling the shrink action "sensitive" with no other explanation will be a little confusing here. We should explain why there are situations where we cannot stop immediately.

Reply:
I think in this table we could not mention the shrink action and just say `ILM has received a request to stop but is still processing some policies`? We can explain why in the stopping ILM section below

Reply:
++ Colin

Reply:
I will punt on this until we have more details on a per-action basis. I think we can link to a description of "safe" actions and have each action's documentation detail what is safe to do while it is running